internal/tss/channel.go

```diff
@@ -0,0 +1,34 @@
+package tss
+
+type RPCGetIngestTxResponse struct {
+	// a status that indicated whether this transaction failed or successly made it to the ledger
+	Status string
+	// the raw TransactionEnvelope XDR for this transaction
+	EnvelopeXdr string
+	// the raw TransactionResult XDR of the envelopeXdr
```

Comment:
Let's try to stay consistent with the formatting across the codebase
```suggestion
	// The raw TransactionResult XDR of the envelopeXdr
```

Reply:
Sounds good